src/Main.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        boolean play = true;
+        String ongeldig = "Ongeldige invoer";
+
+
+
+        while (play) {
+            Scanner scanner = new Scanner(System.in);
+
+            Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+            String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
```

Comment:
Het aanmaken van de lists is handiger om buiten de while loop te zetten, waardoor je programma efficienter wordt. Dit hoeft namelijk maar 1 keer te gebeuren. Het zelfde geldt voor de scanner.